### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by PadowYT2**
*From #2759*

### CODE REVIEW

File: custom/templates/DefaultRevamp/js/core/user.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ if (loggedIn == 1) {
 	var countPms = 0;
 	var countAlerts = 0;
 
-    function updateAlerts(data) {
+	function updateAlerts(data) {
```
Comment (Derkades) [inline custom/templates/DefaultRevamp/js/core/user.js:10]: Did you mean to use tabs here? Nameless PHP code should be using 4 spaces. Everything else looks good.

Comment (PadowYT2) [reply custom/templates/DefaultRevamp/js/core/user.js:10]: Well blame eslint, should I make it a 4 space?

**Comment by Derkades** [comment]
What settings did you use? Some template files seem to use 2 spaces, while others use 4 spaces

**Comment by Derkades** [comment]
Thanks!

**Pull request merged**